### DISCUSSION TITLE
Albums empty after resetting index

### DISCUSSION THREAD

**Discussion by hkelly93** [photoprism/photoprism#3500] [Q&A]
Hi,

I had an issue with getting photos to show up, so I ran `photoprism reset` and only reset the index. I'm currently rerunning it, but some photos that are indexed aren't showing up in albums.

Do I have to go through and recreate all of my albums again? I didn't clear out any of the config files when I ran `photoprism reset`, I expected everything to show up after I indexed again.

**Comment by lastzero** [comment]
You could have forced a full rescan instead? Depending on if you also deleted the YAML sidecar files, your photos may now have different unique IDs so the reference to the albums is broken.

**Reply by hkelly93** [reply]
I tried that multiple times and it wouldn’t show them. I didn’t delete any YAML, only the index.

**Reply by hkelly93** [reply]
I have a backup of my VM that photoprism is on, I restored that and I'm trying a full rescan. I `cd`ed into the `originals` directory and ran `photoprism index -f`. I'll let you know what happens.

**Reply by lastzero** [reply]
To clarify, deleting the index deletes ALL database contents including the albums - this should be mentioned when you run the command. However, albums can be restored from the YAML sidecar files, but only if the photos have the same UIDs (for which other YAML sidecar files exists). Please see our documentation for details:
- https://docs.photoprism.app/user-guide/backups/export/
